qiskit/exceptions.py

```diff
@@ -61,6 +61,24 @@
 is not present, but will raise :exc:`OptionalDependencyImportWarning` to let you know about it.
 
 .. autoexception:: OptionalDependencyImportWarning
+
+When experimental features are being used, Qiskit will raise :exc:`ExperimentalWarning`.
```

Comment:
Maybe this is a good spot to warning about the risk of using these features. Something along these lines:

```suggestion
When experimental features are being used, Qiskit will raise :exc:`ExperimentalWarning`.

.. warning::
    Qiskit experimental features can break at any minor release and their API might change without previous notification. Their use is not recommended in production.
```